File: Linked Lists/001.cpp
#include <iostream>

// #include <bits/stdc++.h>
// #include <stdio.h>
using namespace std;
class Node {
    public:
    int data;
    Node* next;
};
void printList(Node* n){
while(n!=NULL){
    cout<<n->data<<" ";
    n=n->next;
}
}
int main(){
    Node* head=NULL;
    Node* second=NULL;
    Node* third=NULL;

    head = new Node();
    second=new Node();
    third= new Node();

    head->data=1;
    head->next=second;

    second->data=3;
    second->next=third;

    third->data=7;
    third->next=NULL;
    printList(head);
return 0;
}

File: Linked Lists/001.py
class Node:
    def __init__(self,data):
        self.data = data
        self.next=None
class LinkedList:
    def __init__(self):
        self.head=None
    def __printList__(self):
        temp=self.head
        while(temp):
            print(temp.data)
            temp=temp.next
if __name__=='__main__':
    llist=LinkedList()
    llist.head=Node(2)
    first=Node(4)
    second=Node(6)

    llist.head.next=first
    first.next=second
    llist.__printList__()
    